example/src/App.js

```diff
@@ -52,6 +52,10 @@ const LogoImage = styled("img")`
   margin-right: 1em;
 `;
 
+function getBackEndUrl(urlString) {
+  return (!urlString) ? "" : new URL(urlString, window.location.href).toString();
+}
```

Comment:
I get that ternaries are good down in the like actual JSX bits but here, I think it's too clever. Expanding this lets us add comments for each bit, which I think makes this non-obvious logic more clear.

```suggestion
function getBackEndUrl(urlString) {
  // If given something falsey, we need to also return something falsey.
  // Then ensures that we don't accidentally for the application to always be
  // in editor/REST-mode.
  if (!urlString) {
    return null;
  }
  // Because the URL may be a relative path and various application logic
  // assumes that it will always be an absolute URL, it is easiest to just
  // handle converting the potentially relative URL to an absolute URL and
  // explicitly use the current origin as the base.
  try {
    return new URL(urlString, window.location.href).toString();
  } catch (err) {
    // If the given URL is invalid, for some reason, fallback to just using
    // the provided value. This may result in more predictable errors
    // elsewhere.
    console.log(`Unable to create an absolute URL for ${urlString}`);
    return urlString;
  }
}
```